app/web/components/Dialog.tsx

```diff
@@ -54,7 +54,7 @@ export interface AccessibleDialogProps extends Omit<DialogProps, "className"> {
 }
 
 export function Dialog(props: AccessibleDialogProps) {
-  return <MuiDialog {...props} fullWidth maxWidth="sm" scroll="body" />;
+  return <MuiDialog {...props} fullWidth maxWidth="md" scroll="body" />;
```

Comment:
I think it looks better with `sm`

Reply:
Will change back. I thought the default changed with the MUI changes but maybe not for the maxWidth.

It's confusing because for anything `theme.breakpoints.down` we needed to change the value to keep it the same.

Reply:
Ah I see

Reply:
Okay dialog is changed back now. As were the others that weren't related to signup or login.